### PR TITLE
Add util function to drop_collection

### PR DESCRIPTION
**Pull request by XuanYang-cn**
- Only expose `connections` and `Connections`
- Mend tests
- Add docs for drop_collections

Resolves: #652, #654

Signed-off-by: XuanYang-cn <xuan.yang@zilliz.com>

### CODE REVIEW

File: pymilvus/orm/tests/utils.py [hunk 1]
```diff
@@ -42,23 +43,23 @@ def binary_support():
     return ["BIN_FLAT", "BIN_IVF_FLAT"]
 
 def gen_collection_name():
-    return f'ut-collection-' + str(random.randint(100000, 999999))
+    return f'ut_collection_' + str(random.randint(100000, 999999))
```
Comment (scsven) [inline pymilvus/orm/tests/utils.py:46]: Why do we need this changes?

Comment (XuanYang-cn) [reply pymilvus/orm/tests/utils.py:46]: These names are illegal for containing "-", we only support "_"

Comment (XuanYang-cn) [reply pymilvus/orm/tests/utils.py:46]: BTW, the unit tests are still unrunnable, needs further amendment.

**Pull request merged**